src/Momento.Sdk.Incubating/Internal/ScsDataClient.cs

```diff
@@ -128,19 +129,19 @@ public async Task<CacheSetBatchResponse> SendSetBatchAsync(IEnumerable<Task<Cach
     private _DictionaryFieldValuePair[] ToSingletonFieldValuePair(string field, string value) => new _DictionaryFieldValuePair[] { new _DictionaryFieldValuePair() { Field = field.ToByteString(), Value = value.ToByteString() } };
     private _DictionaryFieldValuePair[] ToSingletonFieldValuePair(string field, byte[] value) => new _DictionaryFieldValuePair[] { new _DictionaryFieldValuePair() { Field = field.ToByteString(), Value = value.ToByteString() } };
 
-    public async Task<CacheDictionarySetResponse> DictionarySetAsync(string cacheName, string dictionaryName, byte[] field, byte[] value, bool refreshTtl, TimeSpan? ttl = null)
+    public async Task<CacheDictionarySetResponse> DictionarySetAsync(string cacheName, string dictionaryName, byte[] field, byte[] value, CollectionTtl ttl = default(CollectionTtl))
```

Comment:
@kvcache this is as close as I could get because C# won't let you use a default argument that is not a compile-time constant.  So your `static readonly FROM_CACHE` idea didn't work.  :(

Reply:
ugh i always forget. It seems fine to me though to use `default`. I think this is nicer than nullable. It at least still tells people that something is unconditionally done w.r.t. ttl on every request.